src/main/java/frc/team4909/robot/subsystems/elevator/commands/SetElevatorPosition.java

```diff
@@ -1,19 +1,28 @@
 package frc.team4909.robot.subsystems.elevator.commands;
 
+import edu.wpi.first.wpilibj.command.Command;
 import edu.wpi.first.wpilibj.command.InstantCommand;
 import frc.team4909.robot.subsystems.elevator.ElevatorSubsystem;
 import frc.team4909.robot.Robot;
 
 public class SetElevatorPosition extends InstantCommand {
```

Comment:
there's your issue. instant commands are designed to immediately exit, that would cause the OI control to take precedence